pkg/integrations/metrics/prometheus/tenant.go

```diff
@@ -126,4 +129,28 @@ var (
 		Name: string(TenantReassignedTasksTotal),
 		Help: "The total number of tasks that were reassigned to a worker",
 	}, []string{"tenant_id"})
+
+	TenantWorkerSlots = promauto.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: string(TenantWorkerSlotsTotal),
+			Help: "Total number of slots per worker pool",
+		},
+		[]string{"worker_id", "worker_name"},
```

Comment:
We should add the `tenant_id` label here which is used by the prom metrics endpoint to filter over.

Reply:
nice, thanks! will fix that